### PR TITLE
address errataum 7888

### PR DESCRIPTION
**Pull request by seanturner**
Fixes [erratum 7888](https://www.rfc-editor.org/errata_search.php?eid=7888).

### CODE REVIEW

File: draft-ietf-lamps-rfc4210bis.md [hunk 1]
```diff
@@ -2436,7 +2436,7 @@ See {{sect-5.2.1}} and {{RFC4211}} for CertReqMessages syntax.
 ### Initialization Response
 {: id="sect-5.3.2"}
 
-An Initialization response message contains as the PKIBody an
+An Initialization response message contains as the PKIBody a
 CertRepMessage data structure, which has for each certificate
```
Comment (seanturner) [inline draft-ietf-lamps-rfc4210bis.md:2440]:
```suggestion
An Initialization response message contains a
CertRepMessage data structure in PKIBody, which has for each certificate
```

Comment (HBrock) [reply draft-ietf-lamps-rfc4210bis.md:2440]: The fixing of the erratum  (an -> a) here is fine.

But, PKIBody is a choice and I am wondering if 'as' is correct.
But I am not a native speaker, so I will follow your advice :-)
If we change the text here, we also need to change it in Sections 5.3.1, 5.3.3, and 5.3.4.

Comment (HBrock) [reply draft-ietf-lamps-rfc4210bis.md:2440]: @seanturner should I only change the a -> an or should I also implement the change you proposed to Section  5.3.1, 5.3.3, and 5.3.4?

**Comment by HBrock** [comment]
The erratum was fixed with V10 of the draft

**Pull request closed without merging**